ouroboros-network-framework/src/Data/Wedge.hs

```diff
@@ -0,0 +1,45 @@
+{-# LANGUAGE DeriveFunctor  #-}
+{-# LANGUAGE DeriveFoldable #-}
+
+module Data.Wedge where
```

Comment:
Why not use `Data.Wedge` from the existing Hackage library?

Reply:
trying to minimise dependencies ... am I gone too far? We probably anyway transitively depend on `template-haskell` somewhere else.

Reply:
I didn't realize Data.Wedge used `template-haskell`, it was more of a curiosity question than anything. I think minimizing dependencies  for this case is rather simple 😄